src/lib/email.service.ts

```diff
@@ -1,23 +1,26 @@
 import * as mandrill from 'mandrill-api';
-import * as config from '../config/client-config';
+import * as _ from 'lodash';
+import { getMandrillApiKey }  from '../config/client-config';
 import { mapTemplateEmail } from '../mapper/templateMapper';
-import { ITemplateRequest } from './goodModels/ITemplateRequest';
+import { ITemplateRequest } from './models/ITemplateRequest';
+import { IMandrilTemplateRequest } from './models/IMandrilTemplateRequest';
 
-if (config.getMandrillApiKey() === '') throw new Error('No Mandrill api key provided');
-const client = new mandrill.Mandrill(config.getMandrillApiKey(), process.env.LOG_LEVEL === 'debug');
+export async function sendEmailWithTemplate(mailInfo: ITemplateRequest, mandrillOptions?: any) {
```

Comment:
If you set mandrillOptions to `= {}` as default, you do not have to provide the
`
if (mandrillOptions) {
`
check